pera-execution/latest/pera-adapter/src/execution_engine.rs

```diff
@@ -297,6 +300,7 @@ mod checked {
 
         // We must charge object read here during transaction execution, because if this fails
         // we must still ensure an effect is committed and all objects versions incremented
+
         let result = gas_charger.charge_input_objects(temporary_store);
```

Comment:
???

Reply:
This is not a code that was changed or created by us, please refference only greed or red lines.

Reply:
I think Erik is referencing the fact that an empty line was added between the comment and the let statement.